source/standalone/workflows/ray/isaac_ray_tune.py

```diff
@@ -0,0 +1,335 @@
+                            * spec.config.runner_args["--num_envs"]
+                            * spec.config.runner_args["--num_envs"]
+                        )
+                    )
+                ),
+            )
+        if vary_cnn:
+
+            def generate_cnn_layer():
+                return {
+                    "filters": tune.randint(2**4, 2**9 + 1),
+                    "kernel_size": tune.randint(2**1, 2**4 + 1),
+                    "strides": tune.randint(2**1, 2**4 + 1),
+                    "padding": tune.choice([0, 1]),  # Padding remains as a discrete choice
+                }
+
+            cfg["hydra_args"]["agents.params.network.cnn"] = {
+                "type": "conv2d",
+                "activation": tune.choice(["relu", "tanh", "sigmoid"]),
+                "initializer": {"name": tune.choice(["default", "he_uniform", "glorot_uniform"])},
+                "regularizer": {
+                    "name": tune.choice([None, "l2", "l1"]),
+                },
+                "convs": [generate_cnn_layer() for _ in range(tune.randint(1, 6).sample())],
+            }
+
+        if vary_mlp:
+
+            def generate_mlp_layer():
+                return {
+                    "units": tune.randint(2**3, 2**12),
+                    "activation": tune.choice(["relu", "tanh", "sigmoid", "elu"]),
+                    "initializer": {"name": tune.choice(["default", "he_uniform", "glorot_uniform"])},
+                }
+
+            cfg["hydra_args"]["agents.params.network.mlp"] = {
+                "layers": tune.sample_from(
+                    lambda _: [generate_mlp_layer() for _ in range(tune.randint(1, 10).sample())]
+                )
+            }
+        super().__init__(cfg)
+
+
+class RLGamesResNetCameraJob(RLGamesCameraJobCfg):
+    def __init__(self, cfg: dict = {}):
+        cfg = isaac_ray_util.populate_isaac_ray_cfg_args(cfg)
+        cfg["hydra_args"]["env.observations.policy.image.params.model_name"] = tune.choice(
+            ["resnet18", "resnet34", "resnet50", "resnet101"]
+        )
+        super().__init__(cfg, vary_env_count=True, vary_cnn=False, vary_mlp=True)
+
+
+class RLGamesTheiaCameraJob(RLGamesCameraJobCfg):
+    def __init__(self, cfg: dict = {}):
+        cfg = isaac_ray_util.populate_isaac_ray_cfg_args(cfg)
+        cfg["hydra_args"]["env.observations.policy.image.params.model_name"] = tune.choice([
+            "theia-tiny-patch16-224-cddsv",
+            "theia-tiny-patch16-224-cdiv",
+            "theia-small-patch16-224-cdiv",
+            "theia-base-patch16-224-cdiv",
+            "theia-small-patch16-224-cddsv",
+            "theia-base-patch16-224-cddsv",
+        ])
+        super().__init__(cfg, vary_env_count=True, vary_cnn=False, vary_mlp=True)
+
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser(description="Tune Cartpole.")
+    parser.add_argument("--tune_type", choices=["standard_no_tune", "standard", "resnet", "theia"])
```

Comment:
Can you add  `help` arg for this one?

Reply:
forgot to delete this arg not used just did